modules/web/middleware/binding.go

```diff
@@ -136,7 +136,14 @@ func Validate(errs binding.Errors, data map[string]interface{}, f Form, l transl
 			case validation.ErrRegexPattern:
 				data["ErrorMsg"] = trName + l.Tr("form.regex_pattern_error", errs[0].Message)
 			default:
-				data["ErrorMsg"] = l.Tr("form.unknown_error") + " " + errs[0].Classification
+				msg := errs[0].Classification
+				if msg == "" {
+					msg = errs[0].Message
+				}
```

Comment:
```suggestion
				if msg == "" {
					msg = errs[0].Message
				} else if errs[0].Message != "" {
					msg += ": " + errs[0].Message
				}
```

Maybe the `Message` could be still shown to users even if there is a `Classification`. I guess there will be more information in the `Message` in most cases.

Reply:
Sure (I initially didn't want to change to much logic here)

Reply:
done in d70c1042c